### ISSUE TITLE
#follow doesn't limit to render distance

### ISSUE THREAD

**Issue by 265866** [cabaletta/baritone#2596]
## Describe your suggestion
Currently, #follow is limited to entities in your render distance. my suggestion is to make it so its limited by another setting, #set followDistance (integer). This would allow you to follow players/entities that are out of render distance.

## Settings
New setting - #set followDistance (integer) - would allow you to set chunk limit that entities would be found im

## Context
This would improve baritone because we could find entities or players out of render distance. The reason I came up with this idea was because I wanted to find foxes, but there were none within 32 chunks - it would also help find players that could be a few thousand blocks away

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

## Other
I really don't know if this would be possible at all, I don't know anything about java, Minecraft or anything related to that, so sorry if this was just a waste of time and not possible

Thank you for reading though.

**Comment by l1ving** [comment]
> This would improve baritone because we could find entities or players out of render distance

That's not possible without caching the position of all entities which Baritone does not do

**Comment by ZacSharp** [comment]
Baritone doesn't limit search to render distance by itself, it simply scans all loaded entities and Minecraft only loads entities in render distance.
In singleplayer it would be possible to work around this by forcing Minecraft to load more chunks than render distance, but it would be easier to just increase render distance instead and neither of those works on servers because,
on servers we have to rely on what the server sends us, if the server doesn't send it, it doesn't exist.

TLDR: no, not possible

But, it would be possible
* to just walk to the last known location of an entity if it left our render distance in order to bring it back into render distance
* or explore randomly to find new entities to follow

10 seconds to slow :(
